shared/teams/main/index.js

```diff
@@ -20,7 +20,7 @@ const Teams = (props: Props) => (
     <Header loaded={props.loaded} onCreateTeam={props.onCreateTeam} onJoinTeam={props.onJoinTeam} />
     <Box style={{flex: 1, position: 'relative', width: '100%'}}>
       <ScrollView
-        style={globalStyles.fillAbsolute}
+        style={{...globalStyles.fillAbsolute}}
```

Comment:
was this because of flow or something?

Reply:
yeah, there's a comment about it [here](https://github.com/keybase/client/blob/master/shared/styles/index.js.flow#L8)